org.freedesktop.Platform.VulkanLayer.MangoHud.yml

```diff
@@ -66,6 +66,10 @@ modules:
           version-query: |
             map(select(.tag_name | startswith("v"))) | first |
             .tag_name | sub("^[vV]"; "")
+        # https://github.com/flightlessmango/MangoHud/commit/dc1761e98a435aaee6a919e21f43b85cc38500ac
+      - type: shell
+        commands:
+          - sed -i "s/, '-static-libstdc++'//" src/meson.build
```

Comment:
This could fail silently when upstream changes the file. I think a patch file is a more appropriate place to make this kind of change to upstream's files.

Reply:
Agree. Even better to submit upstream patch with meson build option.